examples/external-security/build.gradle

```diff
@@ -0,0 +1,13 @@
+plugins {
+	// The properties plugin must be applied BEFORE ml-gradle so that ml-gradle sees the results of the properties plugin
```

Comment:
I think this is documented elsewhere and can be removed - I like to slim these example projects down to the bare minimum so it's clear what's being demonstrated.